samples/dashboard/demos/climate/demo.js

```diff
@@ -72,79 +34,169 @@ async function ajaxAll(requests) {
     return Promise.all(promises);
 }
 
-function createDashboard(worldMapJSON) {
-    var dashboard = new Dashboard.Dashboard('container', {
+function buildDates() {
+    const dates = [];
+
+    for (let date = new Date(Date.UTC(2010, 0, 5)),
+            dateEnd = new Date(Date.UTC(2010, 11, 25));
+        date <= dateEnd;
+        date = date.getUTCDate() >= 25 ?
+            new Date(Date.UTC(2010, date.getUTCMonth() + 1, 5)) :
+            new Date(Date.UTC(2010, date.getUTCMonth(), date.getUTCDate() + 10))
+    ) {
+        dates.push([date.getTime(), 0]);
+    }
+
+    return dates;
+}
+
+function buildDateTicks() {
+    const dates = [];
+
+    for (let date = new Date(Date.UTC(2010, 0, 15)),
+            dateEnd = new Date(Date.UTC(2010, 11, 15));
+        date <= dateEnd;
+        date = new Date(Date.UTC(2010, date.getUTCMonth() + 1, 15))
+    ) {
+        dates.push(date.getTime());
+    }
+
+    return dates;
+}
+
+async function buildDashboard() {
+    const topology = await Promise
+        .resolve('https://code.highcharts.com/mapdata/custom/world.topo.json')
+        .then(fetch)
+        .then(response => response.json());
+
+    const climateTable = await dataPool.getSourceTable(1262649600000);
+
+    const dashboard = new Dashboard.Dashboard('container', {
         components: [{
             cell: 'time-range-selector',
             type: 'Highcharts',
             chartOptions: {
                 chart: {
                     height: '100px',
-                    type: 'timeline'
                 },
                 series: [{
-                    data: (() => {
-                        const years = [];
-                        for (let i = 1951, iEnd = 2011; i < iEnd; ++i) {
-                            years.push(i);
+                    type: 'scatter',
+                    name: 'Timeline',
+                    data: buildDates(),
```

Comment:
Please change the `cursor` to a pointer for the Timeline series so we know it can be clicked.

Reply:
Add to Asana